components/skip-link/src/stories.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+import { storiesOf } from '@storybook/react';
+import { WithDocsCustom } from '@govuk-react/storybook-components';
+import Paragraph from '@govuk-react/paragraph';
+
+import SkipLink from '.';
+
+import ReadMe from '../README.md';
+
+const stories = storiesOf('Navigation/SkipLink', module);
+stories.addDecorator(WithDocsCustom(ReadMe));
+
+class AutoFocussed extends Component {
+  constructor(props) {
+    super(props);
+    // create a ref to store the focusable DOM element
+    this.focusableRef = React.createRef();
+  }
+
+  componentDidMount() {
+    this.focusableRef.current.focus();
+  }
+
+  render() {
+    return (
+      <SkipLink
+        ref={this.focusableRef}
+        focusable
+        {...this.props}
+      />
+    );
+  }
+}
+
+stories.add(
+  'Component default',
+  () => (
+    <div>
+      <Paragraph>
+        The skip link component in this example should automatically focus.
+        If it loses focus, tab to this example to see it.
+      </Paragraph>
+      <AutoFocussed href="#content">Skip to main content</AutoFocussed>
```

Comment:
I think it would be useful to have an example where we're actually linking through to DOM content that exists - i.e. a paragraph with an id below, maybe even showing how SkipLink sits within a standard page structure/markup/component tree.

Also you're providing href="#content" but #content is already the default prop so maybe either provide a custom prop or show it working with the default one.

Reply:
approved other than this

Reply:
cool - yes - will add another paragraph with an appropriate ID

(FYI - a later PR to this will be to add in an instance of `SkipLink` to the `Page` component - as GDS guidance is that it should be present on every page - added that as #600)

Reply:
docs/examples updated